csrc/cpu/compat.h

```diff
@@ -0,0 +1,5 @@
+#ifdef VERSION_GE_1_3
+#define DATA_PTR data_ptr
+#else
+#define DATA_PTR data
+#endif
```

Comment:
Is there a specific reason that got introduced back again? PyTorch 1.3 support is dropped anyway.

Reply:
I think this part of code is added by mistake. My bad.